recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,47 @@
+
+{% set name = "unifunc" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 9eaa196505dabe471b3ac790f3e84770919f53728de2f6f0ecbbb550812fde97
+  path: ..
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - pip
+    - python
+    - setuptools
+  run:
+  - nltk
+  - numpy
+  - python
+  - requests
+
+test:
+  imports:
+    - UniFunc
```

```suggestion
test:
  imports:
    - unifunc
  commands:
    - pip check
  requires:
    - pip
```
If you list the requirements in your _setup.py_, this should make sure that you're not missing any here.
Do you have any tests that you run upstream?

---

I have the setup.py here:
https://github.com/PedroMTQ/UniFunc/blob/main/setup.py

I think it only needs setuptools to run, I assume packages to run the actual tool are not included here?

---

Packages that are listed in `install_requires` are required when running the package.
`setuptools >=42` would therefore be in `setup_requires` and `nltk` etc. in `install_requires`.

Ref: https://packaging.python.org/en/latest/tutorials/packaging-projects/